extensions/amp-social-share/0.1/test/test-amp-social-share.js

```diff
@@ -175,8 +175,8 @@ describes.realWin('amp-social-share', {
       el.implementation_.handleClick_();
```

Comment:
Need to add the data params as the second argument to `getShare` above. See https://github.com/akonchady/amphtml/blob/bf846cb9d983e57b48250c8f0f2000c400aaaa51/extensions/amp-social-share/0.1/test/test-amp-social-share.js#L126

Reply:
Done.